src/views/esp/view_esp.cpp

```diff
@@ -85,6 +85,85 @@ namespace big
 			if (distance < g.esp.box_render_distance[1] && distance > g.esp.box_render_distance[0] && g.esp.box)
 				draw_list->AddRect({esp_x - (62.5f * multplr), esp_y - (175.f * multplr)}, {esp_x - (62.5f * multplr) + (125.f * multplr), esp_y - (175.f * multplr) + (350.f * multplr)}, esp_color);
 
+			if (distance < g.esp.skeleton_render_distance[1] && distance > g.esp.skeleton_render_distance[0] && g.esp.skeleton)
+			{
+				std::vector<PedBones> bone_ids = // size: 18
```

Comment:
Is there a reason this isn't a const or constexpr?

Reply:
Oh yea, was originally planning on making a side-function to grab all the required bone ids, but got lazy and just decided to do it manually. Looking back must've just forgot to class it as a const afterwards. I'll fix it in a min